flink-core/src/test/java/org/apache/flink/testutils/junit/RetryRule.java

```diff
@@ -35,11 +36,11 @@
  * <pre>
  * public class YourTest {
  *
- *     {@literal@}Rule
+ *     {@literal @}Rule
```

Comment:
Update: After quick chat with Ufuk, we decided to leave it as it is. The rendering seems not well defined anyways (different IDE versions / Java versions differ).